etcdctl/ctlv3/command/global.go

```diff
@@ -165,64 +166,16 @@ func mustClient(cc *clientv3.ConfigSpec) *clientv3.Client {
 }
 
 func newClientCfg(endpoints []string, dialTimeout, keepAliveTime, keepAliveTimeout time.Duration, scfg *clientv3.SecureConfig, acfg *clientv3.AuthConfig) (*clientv3.Config, error) {
```

Comment:
Any reason to keep newClienCfg function? With a lot of arguments to pass, it's better to use structs with named arguments over calling a function with a lot of unnamed positional arguments.

Reply:
It's a good question.  Because it's still used by some places in etcdctl. I think it's easier to make the change in one place. This function is just like an adapter.

Another we need a logger to call clientv3.NewClientConfig, and usually etcdctl commands do not need to depend on `go.uber.org/zap`. If we remove this function, and update all places, then we also need to introduce one more dependency on zap.